app/javascript/packs/document-capture.tsx

```diff
@@ -122,10 +122,12 @@ const App = composeComponents(
     {
       sdkSrc: acuantVersion && `/acuant/${acuantVersion}/AcuantJavascriptWebSdk.min.js`,
       cameraSrc: acuantVersion && `/acuant/${acuantVersion}/AcuantCamera.min.js`,
+      passiveLivenessSrc: acuantVersion && `/acuant/${acuantVersion}/AcuantPassiveLiveness.min.js`,
```

Comment:
Idea: What if we `passiveLivenessSrc` is an optional property that we add only when the feature is enabled, so that we only add one new property instead of two?

Reply:
Ah, that's much cleaner, the [commit](https://github.com/18F/identity-idp/pull/9546/commits/9eda892f911522ce1334df7a898cceaf86774235). Thanks!